test-monitor/nitestmonitor.yml

```diff
@@ -632,6 +632,26 @@ definitions:
         type: array
         items:
           $ref: '#/definitions/ResultSortDefinitionObject'
+  AdvancedQuery:
+    title: Advanced Query Object for Test Data
+    type: object
+    properties:
+      filter:
+        description: The query filter, in Dynamic Linq
+        type: string
+        example: (operator == "user1") || ((programName != "MyProgram") && (totalTimeInSeconds < @0))
+      substitutions:
+        description: >-
+          Substitutions to make into the query filter expression. Substitutions in the query expression
```

Comment:
"Makes substitutions in the query filter expression."

Reply:
"Substitutions for the query expression are indicated by non-negative integers that are prefixed with the "at" symbol."